sklearn/linear_model/tests/test_base.py

```diff
@@ -433,20 +433,49 @@ def test_dtype_preprocess_data():
             assert_array_almost_equal(X_norm_32, X_norm_64)
 
 
-def test_rescale_data():
+@pytest.mark.parametrize('order', ['C', 'F'])
+def test_rescale_data_dense(order):
     n_samples = 200
     n_features = 2
 
     sample_weight = 1.0 + rng.rand(n_samples)
     X = rng.rand(n_samples, n_features)
     y = rng.rand(n_samples)
-    rescaled_X, rescaled_y = _rescale_data(X, y, sample_weight)
+    rescaled_X, rescaled_y = _rescale_data(X, y, sample_weight, order=order)
+    if order == 'C':
+        assert rescaled_X.flags['C_CONTIGUOUS']
+        assert rescaled_y.flags['C_CONTIGUOUS']
+    elif order == 'F':
+        assert rescaled_X.flags['F_CONTIGUOUS']
+        assert rescaled_y.flags['F_CONTIGUOUS']
```

Comment:
In most cases, y is a 1D array so it's F and C continuous anyway..

Reply:
I added tests for 2d target y.